.travis.yml

```diff
@@ -3,8 +3,9 @@ language: python
 env:
 - TOX_ENV=py26
 - TOX_ENV=py27
-- TOX_ENV=py33
-- TOX_ENV=py34
+# TODO: re-enable when bravado-core supports py3.x
+#- TOX_ENV=py33
+#- TOX_ENV=py34
```

Comment:
Would you mind opening an issue for this on the bravado_core github issue tracker and linking it here?

It would be good to restore this support before merging into master, but if there are serious blockers, then I could see an argument for holding off. At least knowing what the blockers were would be good.